src/vivarium/framework/population.py

```diff
@@ -279,6 +279,12 @@ def metrics(self, index, metrics):
         metrics['total_population'] = len(untracked)+len(tracked)
         return metrics
 
+    def get_population(self, untracked) -> pd.DataFrame:
```

Comment:
do we want a deafult on this so it filters out untracked unless you specifically request otherwise?

Reply:
Not in the population manager.  Should be explicit here and have the sensible default at the interactive interface level